lib/class-wp-rest-dependencies-controller.php

```diff
@@ -173,14 +173,19 @@ public function prepare_item_for_response( $dependency, $request ) {
 	 *
 	 * @param WP_REST_Request $request Request.
 	 *
-	 * @return bool|true|WP_Error
+	 * @return bool|WP_Error
 	 */
 	public function get_items_permissions_check( $request ) {
-		if ( $this->check_handle( $request['dependency'] ) ) {
+		$check = $this->check_read_permission( $request['dependency'] );
+		if ( is_wp_error( $check ) ) {
+			return $check;
+		} else if ( true === $check ) {
+			return $check;
+		} else if (  current_user_can( 'manage_options' ) ) {
 			return true;
```

Comment:
Would this not be better like this?
```php
$check = $this->check_read_permission( $request['dependency'] );
if ( is_wp_error( $check ) ) {
	return $check;
}
if ( true === $check || current_user_can( 'manage_options' ) ) {
	return true;
}
```
There is no need for `else/if` when we `return`